processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -341,6 +358,145 @@ private void initializeGroupingKeyV2Dimension(
     }
   }
 
+  // builds internal String<->Integer dictionary for DimensionSelectors that return false for nameLookupPossibleInAdvance()
+  private static class DictionaryBuildingStringGroupByColumnSelectorStrategy extends StringGroupByColumnSelectorStrategy
```

Comment:
Moved these and GroupByColumnSelectorPlus into top level classes in a new 'column' package under 'epinephilinae'